src/autopas/selectors/AutoTuner.h

```diff
@@ -25,10 +26,15 @@
 namespace autopas {
 
 /**
- * Automated tuner for optimal iteration performance.
+ * Calls to the iteratePairwise() method are passed through this class for two reasons:
+ * 1. Measuring time of the iteration.
+ * 2. Selecting an appropriate configuration for the pairwise iteration.
  *
- * This class offers an interface to the iteratePairwise method.
- * Internally it chooses the best container, traversal etc for the current simulation.
+ * The tuner can be in one of two states, depending on whether it currently should look for a new optimum,
+ * which is called a tuning phase. During a tuning phase,for each Configuration multiple measurements can be taken,
+ * which are called samples. To reduce noise, the samples for one configuration are then condensed to one value for
+ * the current tuning phase, called evidence. These evidence are handed on to a tuningStrategy, which selects a) what
+ * Configuration to test next and b) which configuration is the best in this tuning phase.
```

Comment:
```suggestion
 * The tuner can be in one of two states. If it currently should look for a new optimum, it is in the 
 * so-called tuning phase. During a tuning phase, for each Configuration, multiple measurements can be taken,
 * which are called samples. To reduce noise, the samples for one configuration are then condensed to one value for
 * the current tuning phase, called evidence. The evidences are handed on to a tuningStrategy, which selects a) what
 * Configuration to test next and b) which configuration is the best in this tuning phase.
 * If it should not look for a new optimum it is not in a tuning phase.
```

The first sentence is awkward:
- depending -> which is called.
- The second state is somehow missing
-> added